flink-java/src/main/java/org/apache/flink/api/java/ExecutionEnvironment.java

```diff
@@ -245,68 +237,6 @@ public JobExecutionResult getLastJobExecutionResult(){
 		return this.lastJobExecutionResult;
 	}
 
-	// --------------------------------------------------------------------------------------------
-	//  Session Management
-	// --------------------------------------------------------------------------------------------
-
-	/**
-	 * Gets the JobID by which this environment is identified. The JobID sets the execution context
-	 * in the cluster or local environment.
-	 *
-	 * @return The JobID of this environment.
-	 * @see #getIdString()
-	 */
-	@PublicEvolving
```

Comment:
Notice that your pull request description mentioned

```
The public API, i.e., is any changed class annotated with @Public(Evolving): (yes / **no**)
```

However, this pull request actually remove some of `@PublicEvolving` methods. I think they are not in used so that we can safely removed them but it worth to follow the required process.

Reply:
I agree. I updated the PR description.